docs/resources/cc_bandwidth_package.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Cloud Connect (CC)"
+---
+
+# huaweicloud_cc_bandwidth_package
+
+Manages a bandwidth package resource of Cloud Connect within HuaweiCloud.  
+
+## Example Usage
+
+```hcl
+variable "project_id" {}
+
+resource "huaweicloud_cc_bandwidth_package" "test" {
+  name           = "demo"
+  local_area_id  = "Chinese-Mainland"
+  remote_area_id = "Chinese-Mainland"
+  interflow_mode = "Area"
```

Comment:
the local area and remove area are the same, why we set `interflow_mode` to **Area**?

Reply:
deleted